doc/roadmap.md

```diff
@@ -1,20 +1,24 @@
 # EaseAgent Roadmap
 
-## Features
-* Focus on observability for Java ecosystem with a service perspective.
-* Support mainstream middlewares, libraries, and components in Java-based microservices.
+## Principles
+
+- **Cloud Native**. Focus on Cloud Native microservice, containers, observability ecosystem perspective.
+- **Good Extensibility**.  Easy to develop a new plugin for mainstream middleware, libraries, and components.
+- **Be Standard**. Must follow the standard technology, protocol, best practice, 
+- **Third-Part Integration**. Easy to integrate with other mainstream open-source software, such as Kafka, Elasticsearch, Prometheus, Zipkin..., etc.
   
-## Trace and Metric
-EaseAgent uses open source components to implement data collection:
-* The trace data format which EaseAgent is compatible with is [Zipkin](https://zipkin.io/pages/data_model.html).
-* [Metrics](https://github.com/dropwizard/metrics) is used to measure the behavior of critical components.
+## Data Collection
+EaseAgent uses open source components to do data collection:
+- **Tracing**. The trace data format which EaseAgent is compatible with is [Zipkin](https://zipkin.io/pages/data_model.html).
+- **Metrics**. [Dropwizard Metrics](https://github.com/dropwizard/metrics) is used to measure the behavior of critical components.
 
 ## Roadmap
-Name | Issue | Description 
---- |--- |---
-Join CNCF Landscape            | | [CNCF Landscape](https://landscape.cncf.io/card-mode?category=observability-and-analysis&grouping=category)
-Join OpenAPM Landscape            | | [OpenAPM landscape](https://openapm.io/landscape)
-Plugin UnitTest Framework | | Unit Test
-Integrate Test Framework | | CI/CD can verify the EaseAgent for each pull request
-Support loading and unloading plugin dynamiclly| | The application does not need to restart to update plugins.
-Dynamic field added by plugins isolating | | When multiple plugins add dynamic field to the same class, they do not need to be aware of each other.
+Tasks    | Issues | Description 
+-------- |------- |-----------
+Join CNCF Landscape        | | [CNCF Landscape](https://landscape.cncf.io/card-mode?category=observability-and-analysis&grouping=category)
+Join OpenAPM Landscape     | | [OpenAPM landscape](https://openapm.io/landscape)
+Plugin UnitTest Framework  | |  Unit Test
+Integrate Test Framework   | |  CI/CD can verify the EaseAgent for each pull request
+Performance Test Report    | |  The CPU/Memory/Netowrk perforamnce report.
+Plugin Dynamiclly Loading  | |  The application does not need to restart to update plugins.
```

Comment:
The feature has a technical barrier, I don't think it could be easily implemented in the current framework. We should really focus on how to prompt the development experience (including Unit Test framework, ide plugin, etc...)

Reply:
Agree with  zhao-kun's point of view